### PR TITLE
docs(schema-history): add usage guide for schema history

### PR DESCRIPTION
**Pull request by aditya-radhakrishnan**
## Checklist
- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [x] Links to related issues (if applicable)
- [x] Tests for the changes have been added/updated (if applicable)
- [x] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [x] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

**Review by shirshanka — APPROVED** [review]
LGTM

File: docs/schema-history.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+You can see here that some of these fields were added at the oldest dataset version, while some were added only at this latest
+version. Some fields were modified at the latest version!
+
+The great thing about this blame view is that you can activate it for any version and see what the most recent changes were
+at that point in time for each field. Similar to what Git blame would show you, but for understanding your schemas better!
+
+## Schema Timeline View
+
+In the future, we plan on adding a timeline view where you can see what changes were made to various schema fields over time
+in a linear fashion. Stay tuned for updates!
```
Comment (jjoyce0510) [inline docs/schema-history.md:38]: Typically we drop the questions + concerns footer into our guides. Check out: https://datahubproject.io/docs/policies/#feedback--questions--concerns

File: docs/schema-history.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# Schema History Guide
+
```
Comment (jjoyce0510) [inline docs/schema-history.md:2]: Let's put this in a section called "Introduction"

## Introduction

File: docs/schema-history.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+As your data evolves over time, with fields getting added or removed, with type changes happening, with your documentation
+and glossary term classifications changing, you may find yourself asking the question: "What did this dataset's schema
+look like three months ago? Six months? What about a year ago?" We're happy to announce that you can now use DataHub to
+answer these questions! We currently surface a Schema Blame view where you can examine what a schema looked like at a
+specific point in time, and what the most recent changes were to each of these fields in that schema.
+
+## Schema Blame View Guide
```
Comment (jjoyce0510) [inline docs/schema-history.md:9]: Let's rename this to something actionable: "Viewing a Dataset's Schema History"

File: docs/schema-history.md (hunk 1 above)
Comment (aditya-radhakrishnan) [reply docs/schema-history.md:38]: Makes sense!

**Review by jjoyce0510 — APPROVED** [review]

**Pull request merged**